tests/fixtures/marc.py

```diff
@@ -54,13 +52,13 @@ def work(self, collection: Collection | None = None) -> Work:
         edition = self._db.edition()
         self._db.licensepool(edition, collection=collection)
         work = self._db.work(presentation_edition=edition)
-        work.last_update_time = utc_now()
+        work.last_update_time = utc_now() - datetime.timedelta(days=1)
```

Comment:
Not clear why setting this to a day before. Is it so that running daily won't skip days because it might be less than a day? Would be helpful to have some explanation here.